code/game/gamemodes/changeling/powers/transform.dm

```diff
@@ -7,17 +7,138 @@
 	req_human = 1
 	max_genetic_damage = 3
 
+//changeling clothing items
+/obj/item/clothing/head/changeling

```

Comment:
Possible solution: use generic "flesh" for all slots except mask, use mask "flesh" item for the mask slot?